src/web/mjs/connectors/ComicFX.mjs

```diff
@@ -50,6 +51,21 @@ export default class ComicFX extends Connector {
         const uri = new URL(chapter.id, this.url);
         const request = new Request(uri, this.requestOptions);
         const data = await this.fetchDOM(request, 'div#lcanv source.img-responsive');
-        return data.map(image => this.getAbsolutePath(image.dataset.src || image, request.url));
+        return data.map(image => {
+            let url = this.getAbsolutePath(image.dataset.src || image, request.url);
+            try {
+                if(/img\.comicfx\.net\/img\/\d+\//.test(url)) {
+                    const encoded = url.split('/').pop().split('.').shift();
+                    url = decodeURIComponent(atob(encoded));
+                }
+            } catch(error) {/**/}
+            url = url
+                // HACK: bypass 'i0.wp.com' image CDN to ensure original images are loaded directly from host
+                .replace(/\/i\d+\.wp\.com/, '')
+                // HACK: bypass 'cdn.statically.io' image CDN to ensure original images are loaded directly from host
+                .replace(/cdn\.statically\.io\/img\//, '')
+                .replace(/\/(w=\d+|h=\d+|q=\d+|f=auto)(,(w=\d+|h=\d+|q=\d+|f=auto))*\//, '/');
+            return url.includes('img.comicfx.net') ? this.createConnectorURI(url) : url;
+        }).filter(url => !url.endsWith('/4WWje.jpg'));
```

Comment:
Add comment on this filter for future reference "removed because ad" ?

Reply:
Thats a good and valid point that i recently overlooked too often ;)
Added constant with expressive name instead of using a magic string.